core/src/main/java/org/frankframework/parameters/AbstractParameter.java

```diff
@@ -248,7 +248,7 @@ public Object getValue(ParameterValueList alreadyResolvedParameters, Message mes
 		String requestedSessionKey;
 		if (tpDynamicSessionKey != null) {
 			try {
-				requestedSessionKey = tpDynamicSessionKey.transform(message.asSource());
+				requestedSessionKey = tpDynamicSessionKey.transformToString(message.asSource());
```

Comment:
Ik heb dit op een aantal plekken gezien, maar dit mag volgens mij ook als volgt genoteerd worden toch? Hoeft nu niet, maar misschien wel handig.
```suggestion
				requestedSessionKey = tpDynamicSessionKey.transformToString(message);
```

Reply:
Dat kan inderdaad op heel veel plaatsen. Ik neem het gelijk mee. (Draai nog een laatste keer unit-tests voordat ik push).